src/_langs/en/fundamentals/performance/critical-rendering-path/page-speed-rules-and-recommendations.markdown

```diff
@@ -47,6 +47,43 @@ code shows how to use `navigator.sendBeacon()` to send data to the server in the
 </script>
 {% endhighlight %}
 
+If you're on Chrome 42 or later, you can use the `fetch()` method to request data asynchronously. This method processes responses with Promises rather than multiple event handlers. Unlike the response to an XMLHttpRequest, a fetch response is a stream object starting in Chrome 43. This means that a call to `json()` also returns a Promise. 
```

Comment:
Hey Joe - developers may be on Chrome 42, but they won't know if their users are on Chrome 42, so instead of phrasing it this way, can we rephrase to something along the lines of...

The new `fetch()` method provides an easy way to asynchronously request data. Because it is not available everywhere yet, you should use feature detection to test for its presence before use. The `fetch()` method processes responses...